.circleci/config.yml

```diff
@@ -236,3 +265,14 @@ workflows:
       - staging-test-with-rebase:
           requires:
             - lint
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
+      - static-analysis-and-no-known-cves
+      - staging-test-with-rebase-xenial`
```

Comment:
is the backtick at the end of `staging-test-with-rebase-xenial` a typo?

Reply:
it is !

Reply:
fixed, good catch !